getting-started/jdbc/README.md

```diff
@@ -42,7 +42,7 @@ This example requires `jq` to be installed on your machine.
     export QUARKUS_DATASOURCE_PASSWORD=postgres
     export ASSETS_PATH=$(pwd)/getting-started/assets/
     export CLIENT_ID=root
-    export CLIENT_SECRET=s3cr3t
+    export CLIENT_SECRET=secret
```

Comment:
nit: I'm fine with this change, however the old `s3cr3t` was quite unique and easy to search for in text ;)

Reply:
I agreed with @dimas-b for the uniqueness, but the string `secret` was used in regression tests. To change it to something else will have a bigger impact on downstream projects.